FSW/src/headers/commandHandling.hpp

```diff
@@ -71,6 +78,7 @@ namespace commandCode {
         EXIT_MAIN_LOOP,                // exits the main loop
 
         // End of list
+        SELF_DESTRUCT,                
```

Comment:
can't believe I forgot to add this functionality earlier, this will be really nice to have